### PR TITLE
Port System.Security.Claims source  comments to Docs

### PR DESCRIPTION
**Pull request by carlossanlop**
…tnet-api-docs

## Summary

Automatically ported all source code triple slash comments found in System.Security.Claims to their dotnet-api-docs element if the comment didn't exist yet. Mostly exceptions.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Comment by carlossanlop** [comment]
@mairaw  please take a look whenever you get a chance.
Adding System.Security owners @bartonjs, @GrabYourPitchforks in case they have any comments.

File: xml/System.Security.Claims/Claim.xml [hunk 1]
```diff
@@ -165,6 +165,7 @@ if (null != principal)
         <param name="other">The security claim.</param>
         <summary>Initializes a new instance of the <see cref="T:System.Security.Claims.Claim" /> class.</summary>
         <remarks></remarks>
+        <exception cref="T:System.ArgumentNullException">If <paramref name="other" /> is <see langword="null" />.</exception>
```
Comment (rpetrusha) [inline xml/System.Security.Claims/Claim.xml]:
```suggestion
        <exception cref="T:System.ArgumentNullException"><paramref name="other" /> is <see langword="null" />.</exception>
```

File: xml/System.Security.Claims/Claim.xml [hunk 2]
```diff
@@ -201,6 +202,7 @@ if (null != principal)
         <param name="subject">The subject that this claim describes.</param>
         <summary>Initializes a new instance of the <see cref="T:System.Security.Claims.Claim" /> class with the specified reader and subject.</summary>
         <remarks></remarks>
+        <exception cref="T:System.ArgumentNullException">If <paramref name="reader" /> is <see langword="null" />.</exception>
```
Comment (rpetrusha) [inline xml/System.Security.Claims/Claim.xml]:
```suggestion
        <exception cref="T:System.ArgumentNullException"><paramref name="reader" /> is <see langword="null" />.</exception>
```

**Comment by mairaw** [comment]
Yes, that's good @carlossanlop. Let me review the rest now!

File: xml/System.Security.Claims/Claim.xml [hunk 3]
```diff
@@ -237,6 +239,7 @@ if (null != principal)
         <param name="subject">The subject that this claim describes.</param>
         <summary>Initializes a new instance of the <see cref="T:System.Security.Claims.Claim" /> class with the specified security claim and subject.</summary>
         <remarks></remarks>
+        <exception cref="T:System.ArgumentNullException">If <paramref name="other" /> is <see langword="null" />.</exception>
```
Comment (rpetrusha) [inline xml/System.Security.Claims/Claim.xml]:
```suggestion
        <exception cref="T:System.ArgumentNullException"><paramref name="other" /> is <see langword="null" />.</exception>
```

File: xml/System.Security.Claims/Claim.xml [hunk 4]
```diff
@@ -1044,6 +1047,7 @@ if (null != principal)
         <param name="userData">The user data to claim.</param>
         <summary>Writes this <see cref="T:System.Security.Claims.Claim" /> to the writer.</summary>
         <remarks></remarks>
+        <exception cref="T:System.ArgumentNullException">If <paramref name="writer" /> is <see langword="null" />.</exception>
```
Comment (rpetrusha) [inline xml/System.Security.Claims/Claim.xml]:
```suggestion
        <exception cref="T:System.ArgumentNullException"><paramref name="writer" /> is <see langword="null" />.</exception>
```

File: xml/System.Security.Claims/ClaimsIdentity.xml [hunk 5]
```diff
@@ -258,6 +258,7 @@
         <summary>Initializes a new instance of the <see cref="T:System.Security.Claims.ClaimsIdentity" /> class from a serialized stream created by using <see cref="T:System.Runtime.Serialization.ISerializable" />.</summary>
         <remarks></remarks>
         <permission cref="T:System.Security.SecurityCriticalAttribute">requires full trust for the immediate caller. This member cannot be used by partially trusted or transparent code.</permission>
+        <exception cref="T:System.ArgumentNullException">If <paramref name="info" /> is <see langword="null" />.</exception>
```
Comment (rpetrusha) [inline xml/System.Security.Claims/ClaimsIdentity.xml]:
```suggestion
        <exception cref="T:System.ArgumentNullException"><paramref name="info" /> is <see langword="null" />.</exception>
```

File: xml/System.Security.Claims/ClaimsIdentity.xml [hunk 6]
```diff
@@ -747,6 +748,7 @@
   
  ]]></format>
         </remarks>
+        <exception cref="T:System.InvalidOperationException">If <paramref name="identity" /> is a <see cref="T:System.Security.Claims.ClaimsIdentity" /> and <see cref="P:System.Security.Claims.ClaimsIdentity.Actor" /> results in a circular reference back to <see langword="this" />.</exception>
```
Comment (rpetrusha) [inline xml/System.Security.Claims/ClaimsIdentity.xml]:
```suggestion
        <exception cref="T:System.InvalidOperationException"><paramref name="identity" /> is a <see cref="T:System.Security.Claims.ClaimsIdentity" /> and <see cref="P:System.Security.Claims.ClaimsIdentity.Actor" /> results in a circular reference back to <see langword="this" />.</exception>
```

File: xml/System.Security.Claims/ClaimsPrincipal.xml [hunk 7]
```diff
@@ -627,6 +627,7 @@ if (null != principal)
         <summary>Creates a new claims identity.</summary>
         <returns>The created claims identity.</returns>
         <remarks></remarks>
+        <exception cref="T:System.ArgumentNullException">If <paramref name="reader" /> is <see langword="null" />.</exception>
```
Comment (rpetrusha) [inline xml/System.Security.Claims/ClaimsPrincipal.xml]:
```suggestion
        <exception cref="T:System.ArgumentNullException"><paramref name="reader" /> is <see langword="null" />.</exception>
```

**Comment by carlossanlop** [comment]
I think this one also looks good! @rpetrusha

**Comment by rpetrusha** [comment]
Yes, this looks good, @carlossanlop. Thanks. I'll merge your PR now.

**Pull request merged**